jetty-jaspi/pom.xml

```diff
@@ -56,6 +41,24 @@
       <groupId>org.eclipse.jetty.orbit</groupId>
       <artifactId>javax.security.auth.message</artifactId>
     </dependency>
+    <dependency>
```

Comment:
do we some eclipse ip agreement?
Otherwise even it's better to use deptMngt in top pom as it's only test dependency and only used here I'm fine with that.

Reply:
@olamy dependency on `javax.security.auth.message` was there before. The others I added are only `test` dependencies, so they should not need IP clearance. @jmcc0nn3ll?